Simplenote/src/main/java/com/automattic/simplenote/NoteEditorFragment.java

```diff
@@ -547,6 +562,17 @@ public void onGlobalLayout() {
         return mRootView;
     }
 
+    /**
+     * Disable the right arrow key when markdown is disabled and the cursor is at the end of the note.
+     * This will avoid going from the edit tab to the preview tab unexpectedly since the view pager
+     * will page to the {@link NoteMarkdownFragment} even is the edit and preview tabs are not visible.
```

Comment:
```suggestion
     * will page to the {@link NoteMarkdownFragment} even if the edit and preview tabs are not visible.
```

I think the "is" should be "if" here.  I just noticed this typographical error in my comment suggestion.  Sorry about that.

Reply:
Thanks for noticing that! 🌟 

This is now done, you can check the change here: f106a2f722d6e678138e417527f3a8ffd54114d2